### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Jaws/NVDA][MAS4.2.1][IntelliJ][Wipro]:Jaws and NVDA are not announcing the content of "Azure Login" dialog 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1494]
**Repro Steps:**

1. Open and Start IntelliJ.
2. Tab to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Interactive" radio button.
4. Tab to and select "Sign in" button.
5. In" Azure Login" Enter the "email or phone number" in the edit box and select Next button.
6. Observe Jaws and NVDA behaviour.

**Actual Result:**
Jaws and NVDA are not announcing the content of "Azure Login" dialog 

**Expected Result:**
Jaws and NVDA should announce the content present in azure sign in dialog

Jaws and NVDA should announce as
Azure Login Dialog
username edit
tab
Password edit
tab
sign in button
tab
Sign in with PIN or smart card instead link

**Platform:**
IntelliJ IDEA 2017.3.2 (Community edition)
Build# IC-173.4127.27
JRE:1.8.0_152-release-1024-b8amd64
JVM: OpenJDK 64 Bit Server VM by Jetbrains.
OSBuild Version: Version 1709(Os Build 16299.192)
![35723655](https://user-images.githubusercontent.com/22956130/36136347-0068aff0-10b6-11e8-89a0-8a9acc243add.jpg)
![azure login dialog-signin](https://user-images.githubusercontent.com/22956130/36136352-04ff091a-10b6-11e8-8cc2-36a8b0f8bf38.jpg)

**Comment by Eskibear** [comment]
duplicate with #1496

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749948
